### ISSUE TITLE
Refine camera intrinsics for only selected camera models

### ISSUE THREAD

**Issue by gnool** [colmap/colmap#1286]
In my setup, I have two groups of camera models from different hardware. I prefer to refine focal length for only one of them, since the other group is well calibrated by other means and also don't have high amount of images to allow Colmap's refinement. Is there a way to refine the focal length of the first camera model, and not the second one?

I am under the impression that REFINE_FOCAL_LENGTH is a global variable that affects all the camera models rather than just one.

**Comment by tsattler** [comment]
Yes, the `refine_focal_length` parameter affects all cameras. It should be fairly easy to modify the source code though as to only refine certain cameras.

**Comment by quitmeyer** [comment]
ahhh dang, im just bumpin this because i have the same challenge. I have two cameras with really precise intrinsics preloaded, but i want to refine the intrinsics for just one extra camera. Any easy way to do this without hacking the source?
